apps/web-giddh/src/app/theme/ng-social-login-module/auth.service.ts

```diff
@@ -90,7 +90,7 @@ export class AuthService {
         });
     }
 
-    public signOut(): Promise<any> {
+    public signOut(): Promise<void> {
```

Comment:
if function has return statement, why are we changing any to void?

Reply:
`void` is provided because `resolve();` is not passing any data when Promise is getting resolved